app/screens/channel_info/index.js

```diff
@@ -35,7 +36,7 @@ import {
 
 import ChannelInfo from './channel_info';
 
-function mapStateToProps(state) {
+function mapStateToProps(state) {//eslint-disable-line complexity
```

Comment:
Can you also remove this comment? We can look into if that warning is actually helpful to keep around, but we shouldn't be disabling them without a good reason